src/logging_unittest.cc

```diff
@@ -637,7 +637,7 @@ static void GetFiles(const string& pattern, vector<string>* files) {
   } while (FindNextFileA(handle, &data));
   BOOL result = FindClose(handle);
   LOG_SYSRESULT(result);
-#else
+#elif !defined(ANDROID)
```

Comment:
it could fix compile error, but would fail in test?
I think CheckFille would fail at CHECK_EQ(files.size(), 1UL).